### PR TITLE
C51-43:Add material icons

### PR DESCRIPTION
**Pull request by swastikpareek**
## Overview 
The PR adds material icons to the code repo.

## Technical Details
* Material icons are added as a node module dependency. Make sure to do `npm install`.
* Material icons url and name are added as variable names inside _variables.scss file (DRY concept)
### Usage 
To add a icon to the HTML document add  class `material-icons` to any element and inside it add the icon name as the innerHTML.

 e.g. 
```html
<h2> <i class="material-icons">face</i> Title </h2>
```
 here the icon that will show up will be a face icon - [see here](https://google.github.io/material-design-icons/#using-the-icons-in-html)
 List of all [available icons](https://material.io/tools/icons/?style=baseline)

### CODE REVIEW

File: scss/components/_icons.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * e.g. <h2> <i class="material-icons">face</i> Title </h2>
+ * To know better refer - https://google.github.io/material-design-icons/#using-the-icons-in-html
+ * To see the list of all available icons - https://material.io/tools/icons/?style=baseline
+ */
+
+.material-icons {
+  @extend %material-icons;
+  
+  /* Rules for sizing the icon. Refer https://google.github.io/material-design-icons/#sizing*/
+  &.md-18 {
```
Comment (deb1990) [inline scss/components/_icons.scss]: All these sizes are not necessary in our case, as our designs are not made with material design in mind. So we will be using custom font sizes according to the design. So please remove these.

File: scss/components/_icons.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * Contains class rules for material icons with different font sizes
+ *
+ * Usage
+ * To add a icon to the HTML document add a <i> tag with class `material-icons`
+ * e.g. <h2> <i class="material-icons">face</i> Title </h2>
+ * To know better refer - https://google.github.io/material-design-icons/#using-the-icons-in-html
+ * To see the list of all available icons - https://material.io/tools/icons/?style=baseline
+ */
+
+.material-icons {
```
Comment (deb1990) [inline scss/components/_icons.scss]: The components folder is for the CiviCase components like `case-card`, `activity-card` etc. Also `material-icons` is not exactly a component in my opinion. Its just a helper class. So this should be part of `_material-icons.scss`.

File: scss/base/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/**
+ * @file
+ * File contains variables used for the app (e.g. path variables and color variables etc.)
+ */
+  
+// Material icons related variables
+$material-font-name: 'Material Icons';
```
Comment (deb1990) [inline scss/base/_variables.scss]: Please add the string, `icon`. 
`$material-icon-font-name:` & `$material-icon-font-url`

Comment (swastikpareek) [reply scss/base/_variables.scss]: Done

**Pull request merged**